data/style.css

```diff
@@ -6,6 +6,7 @@ body {
 	padding: 0 16px ;
 	margin-bottom: 500px ;
 	font-family: sans-serif ;
+	scrollbar-color: white #151515;
 }
 
```

Comment:
```suggestion
::-webkit-scrollbar{
	width:.66vw;
    	background-color: #151515;
    	color: #aba499;
}

::-webkit-scrollbar-corner{
    	background-color: #181a1b;
}

::-webkit-scrollbar-thumb{
    	background-color: #454a4d;
}
```
OK. In this new suggestion, I have made the background scrollbar colour the same colour as the background on based.cooking and I have made it 2/3 of the size to be streamlined aswell.

Reply:
I did not remove the original firefox affecting property made by OP, but this webkit stuff will likely override it on Mozilla.

Reply:
![Screenshot from 2021-04-23 22-10-33](https://user-images.githubusercontent.com/59586093/115930085-c7d00500-a480-11eb-9e2a-4366524e4244.png)

Reply:
> I did not remove the original firefox affecting property made by OP, but this webkit stuff will likely override it on Mozilla.

It doesn't: https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-scrollbar#-webkit-scrollbar.

But do you have the colors for light mode too?

Reply:
The bg of this test site is the same as that on based.cooking . ( #151515 )

Reply:
Yes, but there are 3 other colors, I have no idea where they show up and also no idea why they are different. Isn't a scrollbar supposed to have two colors? One for the thumb and one for the background (#151515)?

Reply:
No but I guess that can be done, erm. If instead of putting the hex code in I used a variable that changes based on the browsers gypsy theme. Can that be done in CSS? I dunno, I'm giving the mantle of that task to someone else I think. 
>It doesn't
Ok, then you may change your colour code to the same as I have it if you want. Or remove your edits in favour of my webkit. It may be good to have it as a redundancy IDK.

Reply:
> Yes, but there are 3 other colors, I have no idea where they show up and also no idea why they are different. Isn't a scrollbar supposed to have two colors? One for the thumb and one for the background (#151515)?
That is the THUMB
::-webkit-scrollbar-thumb{
    	background-color: #454a4d;
}